src/components/NftPreview.vue

```diff
@@ -1,19 +1,27 @@
 <script setup>
+import { computed } from 'vue';
+
 const props = defineProps({
   item: Object,
   size: Number
 });
 
-function handleError(e) {
-  const size = props.size ? props.size * 2 : 256;
+const fallbackUrl = computed(
+  () =>
+    `https://cdn.stamp.fyi/token/${props.item.contractAddress}?s=${
```

Comment:
We can use the Stamp NFT endpoint in future to avoid fallback

Reply:
That PR wasn't merged yet:
https://github.com/snapshot-labs/stamp/pull/28

Reply:
Yep I'll make review there